template/frameworks/ava/ava.config.js

```diff
@@ -1,5 +1,7 @@
 export default {
-  require: [
-    './test/helpers/setup.js'
-  ]
+  require: ['./test/_setup.js'],
+  sources: ['**/*.{js,vue}'],
+  files: ['test/specs/**/*'],
```

Comment:
Note: This will only test the specfiles. for multiple uses of ava, check out [this repo](https://github.com/vinayakkulkarni/nuxt-ava-e2e-unit-testing)